spec/models/manageiq/providers/ibm_cloud/vpc/cloud_manager/refresher_spec.rb

```diff
@@ -10,39 +10,69 @@
 
   it "tests the refresh" do
     2.times do
-      ems.refresh
-      ems.reload
-
-      assert_ems_counts
-      assert_specific_vm
-      assert_specific_cloud_volume_type
-      assert_specific_cloud_subnet
-      assert_vm_labels
+      mgmt = ems
```

Comment:
If you don't want to call this `ems` better to just change the `let()` up on line 4.

Why `mgmt` though?  Is it supposed to be `management`?  If you don't want `ems` would `cloud_manager` or `vpc_cloud_manager` be more obvious?

Reply:
Yes mgmt stands for management. ems gets called on each of the 2 times. So it is good to leave there.  Dunno if it requires a let, but it works.

Reply:
Oh are you trying to avoid multiple invocations of `let(:ems)` ?  Because it is memoized and lazy evaluated so only the first call instantiates it.